P5/Source/Specs/localProp.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?><!-- Copyright TEI Consortium. Dual-licensed under CC-by and BSD2 licences See the file COPYING.txt for details $Date$ $Id$ -->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec ident="localProp" module="gaiji" xml:id="LOCALPROP" xmlns="http://www.tei-c.org/ns/1.0" xmlns:rng="http://relaxng.org/ns/structure/1.0"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+    <gloss versionDate="2018-08-22" xml:lang="en">locally defined property</gloss>
+    <desc versionDate="2018-08-22" xml:lang="en">holds the name and value of a locally defined character (or glyph) property as part of its attributes.</desc>
+    <classes>
+      <memberOf key="att.global"/>
+      <memberOf key="att.gaijiProp"/>
+    </classes>
+    <content>
+      <empty/>
+    </content>
+    <exemplum versionDate="2019-07-01" xml:lang="und">
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <char xml:id="daikanwaU4EBA">
+                <localProp name="name" value="CIRCLED IDEOGRAPH 4EBA"/>
+                <localProp name="entity" value="daikanwa"/>
+                <unicodeProp name="Decomposition_Mapping" value="circle"/>
+                <mapping type="standard">人</mapping>
+            </char>
+        </egXML>
+    </exemplum>
+    <remarks versionDate="2005-01-14" xml:lang="en">
+        <p>No definitive list of local names is proposed. However, the name <ident>entity</ident> is recommended as a means of naming the property identifying the recommended
```

Comment:
```suggestion
        <p>No definitive list of local names is proposed. However, the name <val>entity</val> is recommended as a means of naming the property identifying the recommended
```
??
After all, it is the suggested value of the `@name` attribute.

Reply:
I wonder if that first sentence should be changed from passive to active voice:
“These Guidelines do not propose a definitive list of local names. However, …”